src/pages/challenges.tsx

```diff
@@ -59,29 +67,38 @@ const Challenges: FC = () => {
                 Click here to Login
               </a>
             </div>
-          ) : <div><p>Something went wrong! Please contact admin</p></div>)
+          )
         }
+
         {
-        isLoading
-          ? (
-            <p>Loading...</p>
-          ) : (
-            <>
-              {
-                Object.keys(filteredChallenge).length > 0
-                  ? Object.keys(filteredChallenge).map((key) => (
-                    filteredChallenge[key].length > 0
-                    && (
-                    <Accordion open title={key} key={key}>
-                      {renderCardList(filteredChallenge[key], key, user.id)}
-                    </Accordion>
-                    )
+          !isLoading && error && 
+            <div>
+              <p>Something went wrong! Please contact admin
+              </p>
+            </div>
+        }
 
-                  )) : (!error && 'No Challenges Found')
-              }
-            </>
-          )
-      }
+        {
+          isLoading
+            ? (
+              <p>Loading...</p>
+            ) : (
+              <>
+                {
+                  Object.keys(filteredChallenge).length > 0
```

Comment:
you have declared `filteredChallenge` as `[]` on top. This would not give error but better declare it as object.

Reply:
I haven't made this change